server/base/src/main/java/org/apache/accumulo/server/master/balancer/TableLoadBalancer.java

```diff
@@ -47,8 +47,8 @@
 
   private TabletBalancer constructNewBalancerForTable(String clazzName, String table) throws Exception {
     String context = null;
-    if (null != configuration)
-      context = configuration.getTableConfiguration(table).get(Property.TABLE_CLASSPATH);
+    if (null != tabletBalancerEnvironment.getConfiguration())
```

Comment:
Not sure.  It doesn't hurt to keep the check unless we are trying to get away from hiding NPE...